.appveyor/appveyor.bat

```diff
@@ -110,8 +110,14 @@ cargo --version
 
 set RUST_BACKTRACE=1
 
-cargo build -vv %CARGO_MODE%
+cargo test -vv %CARGO_MODE%
 if %ERRORLEVEL% NEQ 0 exit 1
 
-cargo test -vv %CARGO_MODE%
+cargo clean %CARGO_MODE%
+if %ERRORLEVEL% NEQ 0 exit 1
+
+cargo test --all-features %CARGO_MODE%
+if %ERRORLEVEL% NEQ 0 exit 1
+
+cargo test --release --all-features %CARGO_MODE%
```

Comment:
I think this line doesn't match up with how the Windows build is structured, as `%CARGO_MODE%` is either empty or already set to `--release`. I *guess* the correct thing to do instead is to comment in the `Release` configuration line in `.appveyor.yml`.

Reply:
I'll uncomment the Release in appveyor.yml, if thats what you meant.